.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: Release
+
+on:
+  release:
+    types: [released, prereleased]
+
+env:
+  CI: true
+  NODE_OPTIONS: --max_old_space_size=4096
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:13
```

Comment:
We could run this on multiple versions (9,10,11,12) with a [matrix](https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix).

Reply:
Yes, very good point. I initially started with this but the build was slow and decided to drop it. 

Let me add it to things that can be improved in followup issue.

Reply:
I'll do it in this PR. I'll narrow the matrix to two major releases per database.
PG: v9 and v13
Mongo: v3 and v4

What do you think?

Reply:
I see v13 of PG is still in beta so I am not sure if we should bother with v13?